build.sbt

```diff
@@ -102,6 +102,10 @@ lazy val lmCore = (project in file("core"))
       // method open(java.net.URL)java.net.HttpURLConnection in object sbt.librarymanagement.Http does not have a correspondent in current version
       // Was private[sbt] and manually checked to be unused in Zinc or sbt
       ProblemFilters.exclude[DirectMissingMethodProblem]("sbt.librarymanagement.Http.open"),
+      // New methods added to LM API
+      ProblemFilters.exclude[ReversedMissingMethodProblem]("sbt.librarymanagement.ModuleDescriptor.*"),
```

Comment:
We're _probably_ ok adding these two abstract methods today, because there _probably_ isn't any plugin or other user of sbt 1 RC2 that is implementing the public API interface `sbt.librarymanagement.ModuleDescriptor`. But I don't think we should exclude any method added to ModuleDescriptor.

Have you seen https://github.com/typesafehub/migration-manager/issues/183?